pkg/util/proto/openapi.go

```diff
@@ -242,6 +242,21 @@ func (p *Primitive) GetName() string {
 	return fmt.Sprintf("%s (%s)", p.Type, p.Format)
 }
 
+// AnyValue is an arbitrary value (object, array or primitive)
+type AnyValue struct {
+	BaseSchema
+}
+
+var _ Schema = &AnyValue{}
+
+func (m *AnyValue) Accept(v SchemaVisitor) {
+	// A AnyValue accepts any type of value, so it's not necessary to call the SchemaVisitor
```

Comment:
I think that decision should be taken by each individual visitor, what do you think?

Reply:
@pwittrock Do you have an opinion?

Reply:
Yes, it should call a schema visitor.  The visitor may want to do something.

Reply:
OK, that was my initial implementation. Then, since all the visitors' methods were empty, I thought I could remove it, since all visitors should always accept any value. 

I'll add the methods again and also rename this to `ArbitraryValue` or whatever you guys think is a better name.